inc/parametrit.inc

```diff
@@ -429,11 +429,20 @@ elseif ($session == '' and $login != 'yes' and (!isset($verkkokauppa) or $verkko
       //haetaan browserin oletuskieli
       $browkieli = substr($_SERVER["HTTP_ACCEPT_LANGUAGE"], 0, 2);
 
-      if ($browkieli == 'sv') {
-        $browkieli = 'se';
+      // Katsotaan onko avainsanaan annettu vaihtoehtoinen login kieli
+      $query = "SELECT selite
+                FROM avainsana
+                WHERE laji = 'LOGIN_KIELI'
+                LIMIT 1";
+      $result = pupe_query($query);
```

Comment:
Tätä ei voi tehdä näin. Nyt tulee jonkun yrityksen oleutskieli. Login vaiheessa ei tiedetä mikä yritys on kyseessä, joten ei voida tallentaa tätä tietoa kantaan.